### DISCUSSION TITLE
[Unknown Mod][Castles] Changes to the Castles, Town's income and Creature Growth [Edited]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3001] [Ideas]
**Changes to the Castles, Town's income and Creature Growth.**

**Common changes to all castles types.**
**Bonus added to some buildings to increase the town's income per day:**
The Tavern, the Thieves Guild, the Marketplace and the Shipyard give some Gold every day (except in the Necromancer's Castle where there is no Tavern):
- The Tavern gives 100 Gold per day.
- The Thieves Guild gives 150 Gold per day.
- The Marketplace gives 100 Gold per day.
- The Shipyard gives 400 Gold per day.

**Changes for buildings in the Knight's Castle only.**
- The Farm gives 200 Gold per day.

**Bonus added to the growth of creature:**
The Captain's Quarters give supplementary creatures.

**Original growth (with the Well and the "Horde" Building):**
**Knight castle:**
- Peasant: 22
- Archer: 10
- Pikeman: 7
- Swordman: 6
- Cavalry: 5
- Paladin: 4

**Barbarian castle:**
- Goblin: 20
- Orc: 10
- Wolf: 7
- Ogre: 6
- Troll: 5
- Cyclops: 4

**Sorceress castle:**
- Sprite: 18
- Dwarf: 8
- Elf:	 6
- Druid: 5
- Unicorn: 4
- Phoenix: 3

**Warlock castle:**
- Centaur: 18
- Gargoyle: 8
- Griffin: 6
- Minotaur: 5
- Hydra: 4
- Green Dragon: 3

**Wizard castle:**
- Halfling: 18
- Boar: 8
- Iron Golem: 6
- Roc: 5
- Mage: 4
- Giant: 3

**Necromancer castle:**
- Skeleton: 18
- Zombie: 8
- Mummy: 6
- Vampire: 5
- Lich: 4
- Bone Dragon: 3

**Growth with changes:**
**Knight castle:**
- Peasant: 34 - Captain: +12
- Archer: 18 - Captain: +8
- Pikeman: 12 - Captain: +5
- Swordman: 10 - Captain: +4
- Cavalry: 8 - Captain: +3
- Paladin: 6 - Captain: +2

**Barbarian castle:**
- Goblin: 30 - Captain: +10
- Orc: 18 - Captain: +8
- Wolf: 12 - Captain: +5
- Ogre: 10 - Captain: +4
- Troll: 8 - Captain: +3
- Cyclops: 6 - Captain: +2

**Sorceress castle:**
- Sprite: 26 - Captain: +8
- Dwarf: 14 - Captain: +6
- Elf:	 10 - Captain: +4
- Druid: 8 - Captain: +3
- Unicorn: 6 - Captain: +2
- Phoenix: 4 - Captain: +1

**Warlock castle:**
- Centaur: 26 - Captain: +8
- Gargoyle: 14 - Captain: +6
- Griffin: 10 - Captain: +4
- Minotaur: 8 - Captain: +3
- Hydra: 6 - Captain: +2
- Green Dragon: 4 - Captain: +1

**Wizard castle:**
- Halfling: 26 - Captain: +8
- Boar: 14 - Captain: +6
- Iron Golem: 10 - Captain: +4
- Roc: 8 - Captain: +3
- Mage: 6 - Captain: +2
- Giant: 4 - Captain: +1

**Necromancer castle:**
- Skeleton: 26 - Captain: +8
- Zombie: 14 - Captain: +6
- Mummy: 10 - Captain: +4
- Vampire: 8 - Captain: +3
- Lich: 6 - Captain: +2
- Bone Dragon: 4 - Captain: +1

These are two options:
"Towns: Tavern, Thieves Guild, Marketplace and Shipyard increase Town's income per day"
"Towns: Captain's Quarters give supplementary creatures at start of the week"

**Comment by Iwyrith** [comment]
I've had that problem when trying to balance out Succession Wars mod for heroes 3 which used statistics from heroes 2 and I must say there is no easy way to increase growth of 6th level creatures because the numbers are so small.
